platforms/js/opencv_js.config.py

```diff
@@ -55,8 +55,26 @@
               'BFMatcher': ['isMaskSupported', 'create'],
               '': ['drawKeypoints', 'drawMatches', 'drawMatchesKnn']}
 
-calib3d = {'': ['findHomography', 'calibrateCameraExtended', 'drawFrameAxes', 'estimateAffine2D', \
-                'getDefaultNewCameraMatrix', 'initUndistortRectifyMap', 'Rodrigues', \
-                'solvePnP', 'solvePnPRansac', 'solvePnPRefineLM']}
+calib3d = {
+    '': [
+        'findHomography',
+        'calibrateCameraExtended',
+        'drawFrameAxes',
+        'estimateAffine2D',
+        'getDefaultNewCameraMatrix',
+        'initUndistortRectifyMap',
+        'Rodrigues',
+        'solvePnP',
+        'solvePnPRansac',
+        'solvePnPRefineLM',
+        'projectPoints',
+
+        # cv::fisheye namespace
+        'fisheye_initUndistortRectifyMap',
+        'fisheye_projectPoints',
+    ],
+}
 
 white_list = makeWhiteList([core, imgproc, objdetect, video, dnn, features2d, calib3d])
+
+# namespace_prefix_override['dnn'] = ''  # compatibility stuff (enabled by default)
```

Comment:
Wouldn't it be clearer to define the compatibility behaviour here and leave namespace_prefix_override empty in embindgen.py?

Reply:
This file is a sample of opencv.js configuration.
Users of custom opencv.js builds already have own configurations, so we need guarantee compatibility for them.